### PR TITLE
Bump version to 2.7.1.

### PR DESCRIPTION
**Pull request by troessner**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 2.7.1 / 2016-02-09
+
+* [CHANGE] Bump Reek to 3.10.1 (by Tomasz Pajor)
+* [BUGFIX] Analyse only the files whose paths are specified in the CLI (by Nuno Silva)
```
Comment (troessner) [inline CHANGELOG.md]: Historically I can see in the changelog that we never mention the core team. 
I don't see any reason to do so but i see a gazillion reason to change this.
Let's get into the habit of mentioning ourself in the changelog for the future ;)

**Comment by troessner** [comment]
cc @nijikon

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+
 # 2.7.0 / 2015-01-23
```
Comment (Onumis) [inline CHANGELOG.md]: Please take the change and fix there dates!

File: CHANGELOG.md (hunk 1 above)
Comment (Onumis) [inline CHANGELOG.md]: Yeah, why not? :)

File: CHANGELOG.md (hunk 2 above)
Comment (troessner) [inline CHANGELOG.md]: Good point, fixed ;)

**Pull request merged**